src/vellum/core/context.py

```diff
@@ -0,0 +1,82 @@
+from contextlib import contextmanager
```

Comment:
[blocking] we need to avoid putting stuff in `vellum/core` bc Fern auto generates into that repo and will cause conflicts. would suggest something within the `vellum/workflows` directory